src/Nri/Ui/Text/Quiz/V1.elm

```diff
@@ -0,0 +1,67 @@
+module Nri.Ui.Text.Quiz.V1
+    exposing
+        ( footnote
+        , styles
+        )
+
+{-| Text types for quizzes:
+
+@docs footnote
+
+@docs styles
+
+-}
+
+import Css exposing (..)
+import Css.Foreign exposing (Snippet, children, descendants, everything, selector)
+import Html exposing (..)
+import Nri.Colors exposing (..)
+import Nri.Fonts exposing (quizFont)
+import Nri.Ui.Styles.V1
+
+
+{-| This is a little note or footnote.
+-}
+footnote : List (Html msg) -> Html msg
+footnote content =
+    p [ class footnoteClasses ] content
+
+
+footnoteClasses : List CssClasses
+footnoteClasses =
+    [ Footnote ]
+
+
+namespace : String
+namespace =
+    "Nri-Ui-Text-Quiz-"
+
+
+type CssClasses
+    = Footnote
+
+
+{-| -}
+styles : Nri.Ui.Styles.V1.Styles Never CssClasses msg
+styles =
```

Comment:
❓ Should we make this in Html.Styled to make future css conversion easier? (If now is not the time, that's okay too!)